pkg/controller/controller.go

```diff
@@ -57,6 +59,7 @@ import (
 	"github.com/kubernetes-csi/csi-lib-utils/rpc"
 	snapapi "github.com/kubernetes-csi/external-snapshotter/client/v6/apis/volumesnapshot/v1"
 	snapclientset "github.com/kubernetes-csi/external-snapshotter/client/v6/clientset/versioned"
+	referenceGrantv1alpha2 "sigs.k8s.io/gateway-api/pkg/client/listers/apis/v1alpha2"
```

Comment:
just wondering are we fine to use alphav2 of RefGrants? considering we have v1beta1 avaiable , cant we or better to use it ?

Reply:
+1

Reply:
The ReferenceGrant is not yet in beta. Therefore, I'm using Alpha2.
See the URL below.
https://github.com/kubernetes-sigs/gateway-api

Reply:
@ttakahashi21 isnt this https://github.com/kubernetes-sigs/gateway-api/tree/main/pkg/client/listers/apis/v1beta1  available ?

Reply:
Thank you for letting me know.

Reply:
I was able to use it by adding the following to my go.mod file:
> require sigs.k8s.io/gateway-api v0.5.1-0.20221110191515-2d586cc49ab4